app/controllers/manage_services_controller.rb

```diff
@@ -4,7 +4,7 @@ def show
     # If you change this number you must also change it in the corresponding test file - features/manage_metadata/open_drafts_spec.rb.
     @draft_display_max_count = 5
 
-    @drafts = current_user.drafts.where(draft_type: 'ServiceDraft').where(provider_id: current_user.provider_id)
+    @drafts = Draft.where(draft_type: 'ServiceDraft').where(provider_id: current_user.provider_id)
```

Comment:
Should be able too use `ServiceDraft.where` here (thought this model doesn't exist yet)